src/Stan/Observation.hs

```diff
@@ -125,16 +127,22 @@ prettyShowObservation Observation{..} = unlines $
         sols -> formatWith [italic, green] "Possible solution:" :
             map (" ⍟ " <>) sols
 
-{- | Create a 'Observation' 'Id' from the numerical order and 'Inspection' 'Id'.
+{- | Create a 'Observation' 'Id' in a such way that:
+
+1. 'Id' is stable across multiple runs of @stan@ and doesn't depend on
+other inspections in this file.
+2. 'Id' uniquely identifies 'Observation' location.
 
-The 'Observation' should look like this:
+The 'Observation' 'Id' should look like this:
 
 @
-NUM-STAN-000X-XXXX
+STAN-XXXX-Some.Module.Name-line10-column45
```

Comment:
What do you think if we add `OBS` prefix in front of all id? It would indicate clearly, that this is ID of some Observation?

Reply:
Also, I know some modules that have way to long names, it could be frustrating to look through such IDs then :slightly_frowning_face: What if we would map modules to something more short?

Reply:
Regarding"lin" and "column". I think it could be omitted. Just `11:22` could be enough, what do you think?

Reply:
@vrom911 These are all excellent points!